docs/guides/kafka/failover/index.md

```
@@ -0,0 +1,318 @@
+$ kubectl delete pod -n kf-demo -l app.kubernetes.io/instance=kafka-multinode
+pod "kafka-multinode-0" deleted
+pod "kafka-multinode-1" deleted
+pod "kafka-multinode-2" deleted
+```
+
+As the PetSet brings pods back up one at a time (each reattaching its own PVC), they rejoin the metadata
+quorum once a majority is reachable, elect an active controller, and resync partition data from disk.
+Produce/consume resumes once a partition leader is elected again.
+
+```bash
+$ watch kubectl exec -it -n kf-demo kafka-multinode-0 -- kafka-metadata-quorum.sh --command-config config/clientauth.properties --bootstrap-server localhost:9092 describe --status
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)